modules/dashboard/php/dashboard.class.inc

```diff
@@ -40,15 +40,7 @@ class Dashboard extends \NDB_Form
         $site   = $user->getSiteNames();
 
         $userID     = $user->getUsername();
-        $last_login = $DB->pselectOne(
-            "SELECT MAX(Login_timestamp)
-             FROM user_login_history
-             WHERE Login_timestamp < (SELECT MAX(Login_timestamp)
-                FROM user_login_history
-                WHERE userID=:UserID AND Success='Y')
-             AND userID=:UserID AND Success='Y'",
-            array('UserID' => $userID)
-        );
+        $last_login = $user->getLastLogin($DB)->format('Y-m-d H:i:s');
```

Comment:
Will this cause an error to the effect of "Can't use function 'format' on null"?

Reply:
Yes, good catch. I added a hack to handle null and ensure that it's in the same format as the DB was returning for this PR only if it's set, but I'll fix it properly in the modularized widgets PR because work on this one will end up getting lost in the end anyways